spec.emu

```diff
@@ -102,97 +112,166 @@ contributors: Google, Ecma International
             <td>Name indicating August, e.g. Aug, August, etc.</td>
           </tr>
           <tr>
-            <td>`"september"`</td>
+            <td>`"9"`</td>
             <td>Name indicating September, e.g. Sep, September, etc.</td>
           </tr>
           <tr>
-            <td>`"october"`</td>
+            <td>`"10"`</td>
             <td>Name indicating October, e.g. Oct, October, etc.</td>
           </tr>
           <tr>
-            <td>`"november"`</td>
+            <td>`"11"`</td>
             <td>Name indicating November, e.g. Nov, November, etc.</td>
           </tr>
           <tr>
-            <td>`"december"`</td>
+            <td>`"12"`</td>
             <td>Name indicating December, e.g. Dec, December, etc.</td>
           </tr>
+        </table>
+      </emu-table>
+
+    </emu-clause>
+
+    <emu-clause id="sec-isvalidquartercode" aoid="IsValidQuarterCode">
+      <h1>IsValidQuarterCode ( _code_ )</h1>
+      <emu-alg>
+        1. If _code_ is listed in <emu-xref href="#table-validcodeforquarter"></emu-xref>, then
+          1. Return *true*.
+        1. Else,
+          1. Return *false*.
+      </emu-alg>
+
+      <emu-table id="table-validcodeforquarter">
+        <emu-caption>Codes For Quarter of DisplayNames</emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th>Code</th>
+              <th>Description</th>
+            </tr>
+          </thead>
           <tr>
-            <td>`"q1"`</td>
+            <td>`"1"`</td>
             <td>Name indicating the first quarter in year, e.g. Q1, first quarter, 1st quarter, etc.</td>
           </tr>
           <tr>
-            <td>`"q2"`</td>
+            <td>`"2"`</td>
             <td>Name indicating the second quarter in year, e.g. Q2, second quarter, 2nd quarter, etc.</td>
           </tr>
           <tr>
-            <td>`"q3"`</td>
+            <td>`"3"`</td>
             <td>Name indicating the third quarter in year, e.g. Q3, third quarter, 3rd quarter, etc.</td>
           </tr>
           <tr>
-            <td>`"q4"`</td>
+            <td>`"4"`</td>
             <td>Name indicating the fourth quarter in year, e.g. Q4, fourth quarter, 4th quarter, etc.</td>
           </tr>
+        </table>
+      </emu-table>
+
+    </emu-clause>
+
+    <emu-clause id="sec-isvaliddayperiodcode" aoid="IsValidDayPeriodCode">
+      <h1>IsValidDayPeriodCode ( _code_ )</h1>
+      <emu-alg>
+        1. If _code_ is listed in <emu-xref href="#table-validcodefordayperiod"></emu-xref>, then
+          1. Return *true*.
+        1. Else,
+          1. Return *false*.
+      </emu-alg>
+
+      <emu-table id="table-validcodefordayperiod">
+        <emu-caption>Codes For Day Period of DisplayNames</emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th>Code</th>
+              <th>Description</th>
+            </tr>
+          </thead>
           <tr>
             <td>`"am"`</td>
```

Comment:
This is fine for now, but please leave open an issue to decide whether to use am/pm or 1/2.

Reply:
> This is fine for now, but please leave open an issue to decide whether to use am/pm or 1/2.

will do so once land this PR. Does not make sense for such issue w/o this PR in the spec.